lib/manageiq/ui/classic/engine.rb

```diff
@@ -1,7 +1,16 @@
 require 'rails/all'
-require 'sass-rails'
-require 'coffee-rails'
-require 'patternfly-sass'
+
+if ENV["RAILS_ENV"] != "production" || defined?(Rake)
```

Comment:
Sure, makes sense.  That said, this should hopefully be removed when webpack is introduced, but doesn't mean I shouldn't add the comment.